src/main/java/com/kodcu/service/ElasticBulkService.java

```diff
@@ -87,4 +99,35 @@ public void dropDataSet() {
         }
     }
 
+    /**
+     * Customizations for the document.toJson output.
+     *
+     * http://mongodb.github.io/mongo-java-driver/3.0/bson/codecs/
+     *
+     * @return the toJson encoder.
+     */
+  private Encoder<Document> getEncoder() {
+        Map<BsonType, Class<?>> replacements = new HashMap<BsonType, Class<?>>();
+        ArrayList<Codec<?>> codecs = new ArrayList<>();
+
+        if (config.getElastic().getDateFormat() != null) {
```

Comment:
travis says that `Elastic.java` does not have the dateFormat field including getter/setter.